engine/src/witness/sol.rs

```diff
@@ -55,6 +52,9 @@ struct SolanaFeeTrackingVoter {
 	client: SolRetryRpcClient,
 }
 
+// TODO: decide on a reasonable value for this.
+const MIN_PRIORITIZATION_FEE: u64 = 0;
```

Comment:
@albert-llimos would be good to get your input on whether there's a sensible value to use for this.

Reply:
yes this is fine. Shouldn't it be named `DEFAULT_PRIORITIZATION_FEE` though?

Reply:
True - will rename.